_1327/settings.py

```diff
@@ -193,6 +193,9 @@
 
 STATIC_URL = '/static/'
 
+LOGO_FILE = ""
+FAVICON_FILE = "/static/images/favicon.ico"
```

Comment:
I don't think it is a very great idea to hardcode the path to that static file here. Expecially the `static url` could make a problem. I think we have two solutions:

1. use  `FAVICON_FILE = STATIC_URL + "images/favicon.ico"`, or
2. use the static template processor in our templates like so: `{% static "{{FAVICON_FILE}}" %}` (if s.th. like that can work)

We should do this because those files should always reside in the static folder (since they are static files) and it lets us keep the flexibility to easily change the `STATIC_URL` if we ever want to do that... I personally favour option 2, but I'm not sure whether that can work...

Reply:
the second option does not work.
please review again.